scholia/app/templates/author_curation.html

```diff
@@ -55,11 +49,16 @@ <h2 id="missing-citing-author-items-header">Missing citing author items</h2>
 
 <table class="table table-hover" id="missing-citing-author-items-table"></table>
 
+{% endblock %}
 
-<h2 id="missing-topic-header">Authored works with missing topics</h2>
 
-<table class="table table-hover" id="missing-topic-table"></table>
+{% block wikidata %}
 
+<p>For example, here are authored works by {{q}} that do not have topics. You can improve the data by clicking the link
```

Comment:
The ```{{ q }}``` here needs to be replaced with some logic that pulls the label, similar to https://github.com/WDscholia/scholia/pull/1497/files . Perhaps this should be put into a dedicated function that is available globally via scholia.js .

Reply:
Actually, we already have ```entityToLabel``` in scholia.js for this purpose.

Reply:
I intentionally had the explicit Q number, hoping it would help enforce the mental model of it being the ID of this page, but can be changed

Reply:
Perhaps use ```label (QID)``` then?